examples/src/main/java/com/opengamma/strata/examples/finance/FraPricingExample.java

```diff
@@ -96,24 +95,19 @@ private static void calculate(CalculationRunner runner) {
   //-----------------------------------------------------------------------  
   // create a FRA trade
   private static Trade createTrade1() {
-    Fra fra = Fra.builder()
-        .buySell(BuySell.SELL)
-        .index(IborIndices.USD_LIBOR_3M)
-        .startDate(LocalDate.of(2014, 9, 12))
-        .endDate(LocalDate.of(2014, 12, 12))
-        .fixedRate(0.0125)
-        .notional(10_000_000)
-        .build();
-
-    return FraTrade.builder()
-        .product(fra)
-        .info(TradeInfo.builder()
-            .id(StandardId.of("example", "1"))
-            .addAttribute(TradeAttributeType.DESCRIPTION, "0x3 FRA")
-            .counterparty(StandardId.of("example", "A"))
-            .settlementDate(LocalDate.of(2014, 9, 14))
-            .build())
-        .build();
+    return FraConvention.of(IborIndex.of("USD-LIBOR-3M")).toTrade(
+        TradeInfo.builder()
```

Comment:
Looking at this code, the FRA and FxSwap cases are clearer without conventions ;-( Please revert